DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/EnrolController.cs

```diff
@@ -255,17 +255,15 @@ public IActionResult EnrolDelegateSummary(int delegateId, string delegateName)
         [HttpPost]
         public IActionResult EnrolDelegateSummary(int delegateId)
         {
+            var delegateUserId = User.GetUserIdKnownNotNull();
+            var centreId = User.GetCentreIdKnownNotNull();
             var clockUtility = new ClockUtility();
             var sessionEnrol = multiPageFormService.GetMultiPageFormData<SessionEnrolDelegate>(
                MultiPageFormDataFeature.EnrolDelegateInActivity,
                TempData);
             if (!sessionEnrol.IsSelfAssessment)
             {
-                progressDataService.CreateNewDelegateProgress(delegateId, sessionEnrol.AssessmentID.GetValueOrDefault(), sessionEnrol.AssessmentVersion,
```

Comment:
This code is needed. Please revert this change. We will need to set up and poulate delegateId variable above to pass into it, too.